parsers/jscript.c

```diff
@@ -1877,9 +1949,99 @@ static bool parseStatement (tokenInfo *const token, tokenInfo *const parent, boo
 	deleteToken (method_body_token);
 	vStringDelete(saveScope);
 
+	JSCRIPT_DEBUG_LEAVE();
+
 	return is_terminated;
 }
 
+static bool parseES6Class(tokenInfo *const token,tokenInfo * const parent)
```

Comment:
would be useful (and probably not too hard) to support `var foo = class { … }`, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Class_expressions
Especially unnamed ones that is, as they would break parsing here I think